### PR TITLE
#168131446 Users should get stats of trips made in the last X timeframe

### PR DESCRIPTION
**Pull request by Dkabiswa**
#### What does this PR do?
- Adds the return made trip statistics in a specified time frame to a user
#### Description of Task to be completed?
- Get time frame from the user
- Get made trips in that time frame 
#### How should this be manually tested?
- Git clone this repo and checkout to this branch
- Run `npm install` and then run `npm run db-migrate:dev"`
- Run `npm run db-seed:test` to run the new seeders
- Run `npm run start:dev` to start server
- login user at `http://localhost:3000/api/v1/auth/signin`
```
{
"userEmail": "jamesdoe@gmail.com",
"userPassword": "Root1123#"
}
```
- Visit `http://localhost:3000/api/v1/requests/trip-stats` on postman and do a POST
```
{
"parameter": "months",
"value": 8
}
```
- Review the stats

#### Any background context you want to provide?
- For parameter you can use either "months, years, weeks or days"
#### What are the relevant pivotal tracker stories?
[168131446](https://www.pivotaltracker.com/story/show/168131446)
#### Screenshots (if appropriate)
Screen Shot 2019-10-01 at 10 41 36

#### Questions:
N/A

### CODE REVIEW

**Review by willywax — APPROVED** [review]
Looks good

File: src/database/seeders/20190916205822-demo-request.js [hunk 1]
```diff
@@ -12,6 +12,56 @@ export default {
...
+      {
+        from: 'VICTORIA, EAFRICA',
+        travelDate: ['2019-09-01', '2019-10-01'],
+        returnDate: '2019-06-12',
+        reason: 'i want to checkout the nile crocdiles',
+        status: 'Approved',
+        user: 3,
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
```
Comment (willywax) [inline src/database/seeders/20190916205822-demo-request.js:55]: Just update this seeder. The travelDate is greater than request date

Comment (Dkabiswa) [reply src/database/seeders/20190916205822-demo-request.js:55]: have worked on it

**Review by drayzii — APPROVED** [review]
LGTM

**Review by Paulvitalis200 — APPROVED** [review]

**Pull request merged**